src/Core/src/Platform/Android/JavaObjectExtensions.cs

```diff
@@ -30,5 +33,18 @@ public static bool IsAlive([NotNullWhen(true)] this global::Android.Runtime.IJav
 
 			return !obj.IsDisposed();
 		}
+
+		public static TResult? TryJavaCast<[DynamicallyAccessedMembers (Constructors)] TResult>(this IJavaObject? instance)
+			where TResult : class, IJavaObject
+		{
+			try
+			{
+				return instance.JavaCast<TResult>();
+			}
+			catch
+			{
+				return null;
+			}
+		}
```

Comment:
@jonathanpeppers this feels like a waste of CPU cycles... Is there a way to do this where I am checking and casting if the types are valid?

Instead of me keeping the .NET types and all the dependencies, all I want is the Java interface from Android.